packages/gatsby-plugin-nprogress/src/__tests__/gatsby-browser.js

```diff
@@ -0,0 +1,40 @@
+import NProgress from "nprogress"
+import {
+  onClientEntry,
+  onRouteUpdateDelayed,
+  onRouteUpdate,
+} from "../gatsby-browser"
+
+jest.mock(`nprogress`)
```

Comment:
Can we hoist this to the top (I think jest does this anyways, but it's usually clearer for the future "reader" that this module is mocked if the mocks are first)

In other words, this should be the first line of this file.